OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -66,12 +75,12 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
 		public readonly bool CanHover = false;
 
+		[Desc("List of possible action to take when becoming idle in descending order of priority. Invalid entries (e.g. Hover when CanHover is false) will be skipped.")]
```

Comment:
Am I right to assume that we are working towards removing the `CanHover` flag as soon as the activity merging is done? If not, we should merge `Hover` and `Circle` into a single `Wait` (maybe a different name) and use `CanHover` to determine the behaviour.

Reply:
Actually, this is a bit tricky:

We have one case - D2k carryalls - that (at least currently, can't say yet how it'll look after the activities have been merged) need `CanHover` to work, but also circle on idle.

As for removing it after the activities are merged, that might be tricky as well, since it's used for hovering attacks too, for example.

Not sure what the best approach would be here.